react/src/pages/EndpointListPage.tsx

```diff
@@ -272,14 +279,20 @@ const EndpointListPage: React.FC<PropsWithChildren> = ({ children }) => {
         return date1.diff(date2);
       },
     },
-    {
-      title: t('modelService.DesiredSessionCount'),
-      dataIndex: 'desired_session_count',
-      key: 'desiredSessionCount',
-      render: (desired_session_count) => {
-        return desired_session_count < 0 ? '-' : desired_session_count;
-      },
-    },
+    ...(baiClient.supports('replicas')
```

Comment:
Please use `filterEmptyItem`

Reply:
This is used the same as desired_session_count, so I don't think it needs to be version specific.

Reply:
I just suggested this for better readability, but it's a minor issue so it doesn't need to be reflected.